doc/plutus/tutorials/contract-models.rst

```diff
@@ -2906,8 +2906,8 @@ Now ``prop_Auction`` fails!
 
  .. note::
 
-    The balance change is actually a :hsobj:`Plutus.Contract.Test.ContractModel.Symbolics.SymValue`, not a :hsobj:`Plutus.V1.Ledger.Value.Value`,
-    but as you can see it *contains* a :hsobj:`Plutus.V1.Ledger.Value.Value`, which is all we care
+    The balance change is actually a `Plutus.Contract.Test.ContractModel.Symbolics.SymValue` [TODO broken link], not a :hsobj:`PlutusLedgerApi.V1.Value.Value`,
```

Comment:
Please help: `Plutus.Contract.Test.ContractModel.Symbolics.SymValue` is obsolete. What should I replace it with?

Reply:
Looks like it has been moved to https://github.com/input-output-hk/quickcheck-contractmodel/blob/master/quickcheck-contractmodel/src/Test/QuickCheck/ContractModel/Internal/Symbolics.hs